e2e/scripts/k8sBlockLongHaul.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash -e
+
+test_source_repo=$(pwd)
```

Comment:
I think the name is misleading here, we want pwd of the project, not the script. We mount the project src into rook-infra. then run tests from in there.  Also this is going to go away in future when we dump rook infra and start k8s dind directly in the build container.